jplag.frontend-utils/src/test/java/de/jplag/simple/TestTokenConstants.java

```diff
@@ -0,0 +1,5 @@
+package de.jplag.simple;
+
+public interface TestTokenConstants {
+    int STRING = 2;
+}
```

Comment:
I was a bit confused at the beginning because it contains only one parameter but in itself it is a good idea. :+1: